JSDemos/Demos/DropDownBox/SingleSelection/description.md

```diff
@@ -1,15 +1,15 @@
-The DropDownBox component is an editor that consists of a text field and drop-down content. In this demo, the content is the TreeView and the DataGrid in the single selection mode. 
+The DropDownBox component consists of a text field and drop-down content. In this demo, the content is the TreeView and the DataGrid in the single selection mode. 
 
-The following instructions show how to synchronize the DropDownBox with any other embedded DevExtreme component:
+The following instructions show how to synchronize the DropDownBox with an embedded DevExtreme component:
 
 1. **Specify data sources**    
-The DropDownBox's and embedded component's data sources can be the same or different. If they are different, the embedded component's key field should be present in the DropDownBox's data source.        
+The DropDownBox and its embedded component can use the same or different data sources. If the data sources are different, you need to specify the embedded component's key field in the DropDownBox's data source. If the DropDownBox contains a DataGrid, bind the DataGrid to a store instead of a simple array. For example, an [ArrayStore](/Documentation/ApiReference/Data_Layer/ArrayStore/).     
```

Comment:
```suggestion
The DropDownBox and its embedded component can use the same or different data sources. If the data sources are different, you need to specify the embedded component's key field in the DropDownBox's data source. If the DropDownBox contains a DataGrid, bind the DataGrid to a store instead of a simple array (for example, an [ArrayStore](/Documentation/ApiReference/Data_Layer/ArrayStore/)).     
```